docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+.. _ksql-security:
+
+Securing KSQL
+-------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. In the same way,
+KSQL is configured so that the KSQL servers are authenticated and authorized, and data communication is encrypted when
+communicating with the Kafka cluster. KSQL can be configured for:
+
+- :ref:`SSL for encryption <kafka_ssl_encryption>`
+- :ref:`SASL for authentication <kafka_sasl_auth>`
+- :ref:`HTTPS for Confluent Schema Registry <schema_registry_http_https>`
+
+-------------------------------------------------
+Using KSQL with an Apache Kafka Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interactive KSQL clusters require that the KSQL user has open access to create, read, write and delete topics and
```

Comment:
We should clarify what "the KSQL user" refers to here.  It's not the human typing queries into the CLI. I suppose it's related to the principal or sth like that.

Reply:
`...require that the KSQL client has open...`?

Reply:
I don't know the right terminology here, could you check with @big-andy-coates? I suppose it's the authenticated username ("principal?") that the KSQL server uses.

Reply:
While _principal_ is probably the correct term here, I always find it makes security docs hard to read / understand for none security people.

How about:

`Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user.  Interactive KSQL clusters require that this authenticated user has open access...`